src/topology/vector_bundle/basic.lean

```diff
@@ -361,62 +279,86 @@ begin
-e.to_pretrivialization.linear_map_at b
+protected def linear_map_at (e : trivialization F (π E)) [e.is_linear R] (b : B) : E b →ₗ[R] F :=
+e.to_pretrivialization.linear_map_at R b
+
+variables {R}
 
-lemma coe_linear_map_at (e : trivialization R F E) (b : B) :
-  ⇑(e.linear_map_at b) = λ y, if b ∈ e.base_set then (e (total_space_mk b y)).2 else 0 :=
+lemma coe_linear_map_at (e : trivialization F (π E)) [e.is_linear R] (b : B) :
+  ⇑(e.linear_map_at R b) = λ y, if b ∈ e.base_set then (e (total_space_mk b y)).2 else 0 :=
 e.to_pretrivialization.coe_linear_map_at b
 
-lemma coe_linear_map_at_of_mem (e : trivialization R F E) {b : B} (hb : b ∈ e.base_set) :
-  ⇑(e.linear_map_at b) = λ y, (e (total_space_mk b y)).2 :=
+lemma coe_linear_map_at_of_mem (e : trivialization F (π E)) [e.is_linear R] {b : B}
+  (hb : b ∈ e.base_set) :
+  ⇑(e.linear_map_at R b) = λ y, (e (total_space_mk b y)).2 :=
 by simp_rw [coe_linear_map_at, if_pos hb]
 
-lemma linear_map_at_apply (e : trivialization R F E) {b : B} (y : E b) :
-  e.linear_map_at b y = if b ∈ e.base_set then (e (total_space_mk b y)).2 else 0 :=
+lemma linear_map_at_apply (e : trivialization F (π E)) [e.is_linear R] {b : B} (y : E b) :
+  e.linear_map_at R b y = if b ∈ e.base_set then (e (total_space_mk b y)).2 else 0 :=
 by rw [coe_linear_map_at]
 
-lemma linear_map_at_def_of_mem (e : trivialization R F E) {b : B} (hb : b ∈ e.base_set) :
-  e.linear_map_at b = e.linear_equiv_at b hb :=
+lemma linear_map_at_def_of_mem (e : trivialization F (π E)) [e.is_linear R] {b : B}
+  (hb : b ∈ e.base_set) :
+  e.linear_map_at R b = e.linear_equiv_at R b hb :=
 dif_pos hb
 
-lemma linear_map_at_def_of_not_mem (e : trivialization R F E) {b : B} (hb : b ∉ e.base_set) :
-  e.linear_map_at b = 0 :=
+lemma linear_map_at_def_of_not_mem (e : trivialization F (π E)) [e.is_linear R] {b : B}
+  (hb : b ∉ e.base_set) :
+  e.linear_map_at R b = 0 :=
 dif_neg hb
 
-lemma symmₗ_linear_map_at (e : trivialization R F E) {b : B} (hb : b ∈ e.base_set) (y : E b) :
-  e.symmₗ b (e.linear_map_at b y) = y :=
+lemma symmₗ_linear_map_at (e : trivialization F (π E)) [e.is_linear R] {b : B} (hb : b ∈ e.base_set)
+  (y : E b) :
+  e.symmₗ R b (e.linear_map_at R b y) = y :=
 e.to_pretrivialization.symmₗ_linear_map_at hb y
 
-lemma linear_map_at_symmₗ (e : trivialization R F E) {b : B} (hb : b ∈ e.base_set) (y : F) :
-  e.linear_map_at b (e.symmₗ b y) = y :=
+lemma linear_map_at_symmₗ (e : trivialization F (π E)) [e.is_linear R] {b : B} (hb : b ∈ e.base_set)
+  (y : F) :
+  e.linear_map_at R b (e.symmₗ R b y) = y :=
 e.to_pretrivialization.linear_map_at_symmₗ hb y
 
+variables (R)
+
 /-- A coordinate change function between two trivializations, as a continuous linear equivalence.
   Defined to be the identity when `b` does not lie in the base set of both trivializations. -/
-def coord_change (e e' : trivialization R F E) (b : B) : F ≃L[R] F :=
+def coord_changeₗ (e e' : trivialization F (π E)) [e.is_linear R] [e'.is_linear R] (b : B) :
```

Comment:
the l subscript is usually used for linear maps, not continuous linear maps. Do you think that `coord_changeL` is too ugly? Don't we use suffixes like `clm` or `cle` somewhere in the library?

Reply:
Good point.  I think `coord_changeL` is fine, I'll do that.